sites/public/src/components/applications/ApplicationMultiselectQuestionStep.tsx

```diff
@@ -162,15 +208,32 @@ const ApplicationMultiselectQuestionStep = ({
         <FormBackLink
           url={conductor.determinePreviousUrl()}
           onClick={() => {
-            conductor.setNavigatedBack(true)
-            setPage(page - 1)
+            if (!verifyAddress) {
+              conductor.setNavigatedBack(true)
+              setPage(page - 1)
+              body.current = null
+            }
```

Comment:
Is the expected behavior of the back button to go back to the previous question. For example If I have a preference that has multiple address collections and I click "back" on the verification step it brings me back to the "income" step. Is that expected behavior?

Reply:
Task states that back button has to be blocked on verification step. It works like that in first application step (Not sure if by mistake, as in code it seem like we want to set verificationState to `false`, but as we don't have `custom` prop in `FormBackLink` it just gets url instead of `onClick`)

from adress.tsx
```
  const backFunction = useCallback(() => {
    return verifyAddress ? setVerifyAddress(false) : conductor.setNavigatedBack(true)
  }, [verifyAddress])
```